### PR TITLE
Enable: Participate as team for hackathons

### PR DESCRIPTION
**Pull request by RiyaMathew-11**
The event admin/creator of a hackathon can mandate whether it should be a team event on not. 

This is a short PR that mainly handles mandating a team option for hackathons as mentioned in #69 

- It auto-assigns a participant who creates a team as team lead
- Provides a status check for the team lead.

### CODE REVIEW

**Comment by harshtandiya** [comment]
Is this PR ready for review? It is still in draft state

File: fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+    def after_insert(self):
+        participant_hash = frappe.db.get_value(
+            "FOSS Hackathon Participant",
+            {
+                "participant_email": frappe.session.user,
+                "hackathon_name": self.hackathon_name,
+            },
+            "name",
+        )
+        print(participant_hash)
```
Comment (harshtandiya) [inline fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py]: test print I guess? Let's remove it

File: fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 
-# import frappe
+import frappe
 from frappe.website.website_generator import WebsiteGenerator
 
 
 class FOSSHackathonTeams(WebsiteGenerator):
-    pass
+    def after_insert(self):
+        participant_hash = frappe.db.get_value(
```
Comment (harshtandiya) [inline fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py]: Let's just name it participant

**Review by harshtandiya — CHANGES_REQUESTED** [review]

File: fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py (hunk 2 above)
Comment (RiyaMathew-11) [reply fossunited/hackathon/doctype/foss_hackathon_teams/foss_hackathon_teams.py]: Done

**Review by harshtandiya — APPROVED** [review]

**Pull request merged**